src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -112,6 +111,7 @@ private int Run(string[] args)
                     return 0;
                 }
 
+                Console.Out.WriteLine("Generate serializer file for " + assembly);
```

Comment:
Please use resource instead of hard coded strings. This will cause localization bugs in the future.

BTW, do we have verbosity mode (eg, silent, informational, verbose etc.) to control what we output?